### ISSUE TITLE
Error: "Did you forget to signal async completion? Gulp v4.0.0"

### ISSUE THREAD

**Issue by coolemur** [atomicpages/gulp-download2#4]
When using basic example:

```
const gulp = require('gulp');
const download = require('gulp-download2');

gulp.task('download', () => download('http://example.com/file.jpg').pipe(gulp.dest('build')));
```

with gulp v4.0.0 I get error:

**The following tasks did not complete: taskname
Did you forget to signal async completion?**

**Comment by atomicpages** [comment]
@coolemur the gulp API has changed greatly in 4.0.0 from 3.9.1. I realize you posted this quite awhile ago, but if you have a fix for gulp 4.0.0 I'd happy merge the publish a version that works with gulp 4.x

**Comment by atomicpages** [comment]
Tested with gulp 4 and it no longer appears to be an issue. Closing for now 👍
